### ISSUE TITLE
Use RSS to subscribe to a channel

### ISSUE THREAD

**Issue by ErrorTeaPot** [yattee/yattee#711]
### Guidelines

- [X] I have searched the [issue tracker](https://github.com/yattee/yattee/issues) and I haven't found feature request like this

### Type

Improvement to existing feature

### Describe the feature

Since not all instances allows to use an account and sync subscription it could be interesting to use RSS feeds instead. 
By using it, we could just use RSS to detect new videos and link it to yattee so as to start it. We would have a youtube channel the user have subscribed to, this action save the rss feed and subscribe to it and at any new content we get the info. 

This can provide something more modular allowing easy import/export, sync with maybe already existing rss hosts so as to save the subscribed channels.

**Comment by stonerl** [comment]
There is already a similar feature in Yattee. You can mark a channel as favorite and it will appear on the Home view.

**Comment by ErrorTeaPot** [comment]
Yes but the idea is to be able to connect it to an external RSS host. This would allows to get sync subscriptions with other devices and make the import/export operation easier.
